utils/R/do.conversions.R

```diff
@@ -24,10 +24,20 @@ do.conversions <- function(settings, overwrite.met = FALSE, overwrite.fia = FALS
     
     input.tag <- names(settings$run$input)[i]
     
-    # fia database
+    # IC conversion : for now for ED only, hence the css/pss/site check
     if ((input.tag %in% c("css", "pss", "site")) &&
-        is.null(input$path) && !is.null(input$source) && (input$source == "FIA")) {
-      settings <- PEcAn.data.land::fia.to.psscss(settings, overwrite=overwrite.fia)
```

Comment:
1) I wonder if we should retain a way to call fia.to.psscss until the new workflow is fully functioning? 
2) When fia.to.psscss is out of the workflow it needs to start the formal deprecation process (see Backward compatibility under http://r-pkgs.had.co.nz/release.html)